docs/reference/logger.md

```diff
@@ -0,0 +1,57 @@
+# Logger
+
+## Overview
+Logging is a very important part of Ratify as it provides a way to debug and understand the behavior of the system. Ratify uses [Logrus](https://github.com/sirupsen/logrus) as the logging library.
+
+## Usage
+Ratify exposed a few options for users to configure the behavior of the logger. The logger can be configured through the helm chart values or the configuration file.
+
+### Configuration
+When running Ratify as a K8s add-on, users could configure the logger by setting [helm chart values](../../charts/ratify/values.yaml), e.g.
+```yaml
+logger:
+  formatter: "text"
+  level: "info"
+  requestHeaders:
+    traceIDHeaderName:
+      - "x-trace-id"
+      - "x-ms-correlation-request-id"
+```
+
+### Formatter
+Users could set the formatter to be `text`, `json` or `logstash`. The default value is `text` if it's not set.
+
+`json` and `text` are 2 commonly used formats for logging system. `logstash` is another format that's compatible with [Logstash](https://www.elastic.co/logstash).
+
+### Log Levels
+Users could configure log level by setting `logger.level` to the expected level. The default value is `info`.
+
```

Comment:
I don't see how will this PR solve the feature request from https://github.com/deislabs/ratify/issues/972. @binbin-li  Could you clarify how to differentiate the log sources by using the new logger configuration?

Reply:
This PR addes a log option that specifies the component type, which helps identifying the source of log. E.g. LogOption:         logger.Option{ComponentType: logger.Server},

Reply:
Users could configure log formatter to Json or Logstash and use this component-type as filtering field.